### DISCUSSION TITLE
Any tips for using regions with zoom, especially on mobile?

### DISCUSSION THREAD

**Discussion by jagthedrummer** [katspaugh/wavesurfer.js#3322] [Q&A]
I'm trying to use the regions plugin along with the timeline plugin and zoom feature. On desktop my initial experiments are working well. Just for reference it looks like this:

![CleanShot 2023-11-06 at 15 31 26](https://github.com/katspaugh/wavesurfer.js/assets/58702/f2b2d6e3-bef6-472f-9ebb-4aaed5b4a34f)

On mobile things go really weird when trying to modify or create a region. When you try to drag the region sometimes it works as expected and moves the region over the top of the waveform, sometimes it drags the waveform view itself (taking the region along with it), and sometimes it does a weird thing where it moves the region _and_ resizes it at the same time. When that happens the region usually gets bigger as it's dragged to the right and gets smaller as it's dragged to the left.

I think ideally I'd like to be able to do something like pass a `scrollable: false` option to Wavesurfer to prevent user input from affecting the scroll position of the waveform itself so that any dragging could be assume to be related to either moving or creating a region.

I'm pretty new to trying to use regions, so I wouldn't be surprised if I'm just trying to do something that's ill-advised. If there are any good examples of mobile friendly integration with zoop, minimap & regions I'd love to see them.

**Comment by katspaugh** [comment]
Sounds like there might be event propagation issues in that plugin (when dragging falls through and scrolls). I wonder why it happens intermittently though. Resizing when moving also looks like a bug.

I think an option to temporarily freeze scroll makes sense but I would first try to figure out what causes the issues you described.

Cheers!

**Reply by jagthedrummer** [reply]
Should I create an issue about the event propagation problem?

**Reply by katspaugh** [reply]
Yes, that would be great.
